### PR TITLE
Fix minor bugs

### PR DESCRIPTION
**Pull request by logical-1985516**
Mainly typos in display messages

### CODE REVIEW

File: src/main/java/seedu/hirehub/logic/Messages.java [hunk 1]
```diff
@@ -46,7 +47,8 @@ public static String getErrorMessageForDuplicatePrefixes(Prefix... duplicatePref
      */
     public static String format(Person person) {
         final StringBuilder builder = new StringBuilder();
-        builder.append(person.getName())
+        builder.append("Person(Name: ")
+                .append(person.getName())
                 .append("; Phone: ")
                 .append(person.getPhone())
                 .append("; Email: ")
```
Comment (chiralcentre) [inline src/main/java/seedu/hirehub/logic/Messages.java:54]: Possible violation of feature freeze

File: src/main/java/seedu/hirehub/logic/Messages.java [hunk 2]
```diff
@@ -73,4 +75,16 @@ public static String format(Job job) {
                 + ")";
     }
 
+    /**
+     * Formats the {@code application} for display to the user.
+     */
+    public static String format(Application application) {
```
Comment (chiralcentre) [inline src/main/java/seedu/hirehub/logic/Messages.java:81]: Where will this method be used?

File: src/main/java/seedu/hirehub/logic/commands/AddApplicationCommand.java [hunk 3]
```diff
@@ -29,18 +29,17 @@ public class AddApplicationCommand extends Command {
...
-    public static final String MESSAGE_NO_PERSON_IN_LIST = "This candidate is not in the current list of candidates! ";
-    public static final String MESSAGE_NO_JOB_IN_LIST = "This job has not been created";
-    public static final String MESSAGE_EXCEEDS_VACANCY = "The number of accepted candidates already meets the"
-            + " stipulated vacancy.\n To accept more candidates, the vacancy for the job can be increased via the"
-            + " edit_job command, or change status for existing application(s) to this job via status command to"
-            + " a status other than OFFERED.\n You can retrieve vacancies left via slots_left command";
-
+            + "and the application has already been processed into the list.";
+    public static final String MESSAGE_NO_PERSON_IN_LIST = "This candidate is not in the current list of candidates!";
+    public static final String MESSAGE_NO_JOB_IN_LIST = "This job is not in the current list of jobs!";
```
Comment (chiralcentre) [inline src/main/java/seedu/hirehub/logic/commands/AddApplicationCommand.java:38]: possible violation of feature freeze

File: src/main/java/seedu/hirehub/logic/commands/ClearCommand.java [hunk 4]
```diff
@@ -12,7 +12,7 @@
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
+    public static final String MESSAGE_SUCCESS = "Person list has been cleared!";
```
Comment (chiralcentre) [inline src/main/java/seedu/hirehub/logic/commands/ClearCommand.java:15]: Address book is technically not wrong

File: src/main/java/seedu/hirehub/logic/commands/InitDeleteCommand.java [hunk 5]
```diff
@@ -16,7 +16,7 @@
  * Pushes the program into the state to delete the person at targetIndex.
  */
 public class InitDeleteCommand extends Command {
-    public static final String MESSAGE_SUCCESS = "Delete person. Proceed? (Y/N)";
+    public static final String MESSAGE_SUCCESS = "Delete person selected. Proceed? (Y/N)";
```
Comment (chiralcentre) [inline src/main/java/seedu/hirehub/logic/commands/InitDeleteCommand.java:19]: Possible violation of feature freeze

File: src/main/java/seedu/hirehub/logic/commands/ListApplicationCommand.java [hunk 6]
```diff
@@ -12,7 +12,7 @@ public class ListApplicationCommand extends Command {
 
     public static final String COMMAND_WORD = "list_app";
 
-    public static final String MESSAGE_SUCCESS = "Listed all available applications from the list!";
+    public static final String MESSAGE_SUCCESS = "Listed all available applications from the application list!";
```
Comment (chiralcentre) [inline src/main/java/seedu/hirehub/logic/commands/ListApplicationCommand.java:15]: possible violation of feature freeze

File: src/main/java/seedu/hirehub/logic/commands/ListCommand.java [hunk 7]
```diff
@@ -12,7 +12,7 @@ public class ListCommand extends Command {
 
     public static final String COMMAND_WORD = "list";
 
-    public static final String MESSAGE_SUCCESS = "Listed all persons";
+    public static final String MESSAGE_SUCCESS = "Listed all persons from the person list!";
```
Comment (chiralcentre) [inline src/main/java/seedu/hirehub/logic/commands/ListCommand.java:15]: possible violation of feature freeze

File: src/main/java/seedu/hirehub/logic/commands/ListJobCommand.java [hunk 8]
```diff
@@ -12,7 +12,7 @@ public class ListJobCommand extends Command {
 
     public static final String COMMAND_WORD = "list_job";
 
-    public static final String MESSAGE_SUCCESS = "Listed all available jobs from the list!";
+    public static final String MESSAGE_SUCCESS = "Listed all available jobs from the job list!";
```
Comment (chiralcentre) [inline src/main/java/seedu/hirehub/logic/commands/ListJobCommand.java:15]: possible violation of feature freeze

File: src/main/java/seedu/hirehub/logic/commands/SearchApplicationCommand.java [hunk 9]
```diff
@@ -20,6 +20,8 @@ public class SearchApplicationCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all applications whose attributes match all "
             + "the corresponding specified attributes.\n"
+            + "For email and status, only full words will be matched,\n"
+            + "while for title, partial words will be matched.\n"
```
Comment (chiralcentre) [inline src/main/java/seedu/hirehub/logic/commands/SearchApplicationCommand.java:24]: possible violation of feature freeze

File: src/main/java/seedu/hirehub/logic/commands/SearchJobCommand.java [hunk 10]
```diff
@@ -20,6 +20,8 @@ public class SearchJobCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all jobs whose attributes match all "
             + "the corresponding specified attributes.\n"
+            + "For vacancy, only jobs with the same vacancy will be matched,\n"
+            + "while for title and description, partial words will be matched.\n"
```
Comment (chiralcentre) [inline src/main/java/seedu/hirehub/logic/commands/SearchJobCommand.java:24]: possible violation of feature freeze

**Review by chiralcentre — COMMENTED** [review]
Please check if every area I mentioned is viable to be added during v1.4

File: src/main/java/seedu/hirehub/logic/commands/SearchJobCommand.java (hunk 10 above)
Comment (logical-1985516) [reply src/main/java/seedu/hirehub/logic/commands/SearchJobCommand.java:24]: Indeed, as this seems to be making the message usage more specific. Will revert, thanks!

File: src/main/java/seedu/hirehub/logic/commands/SearchApplicationCommand.java (hunk 9 above)
Comment (logical-1985516) [reply src/main/java/seedu/hirehub/logic/commands/SearchApplicationCommand.java:24]: Indeed, as this seems to be making the message usage more specific. Will revert, thanks!

File: src/main/java/seedu/hirehub/logic/commands/ClearCommand.java (hunk 4 above)
Comment (logical-1985516) [reply src/main/java/seedu/hirehub/logic/commands/ClearCommand.java:15]: Will revert, thanks

**Pull request closed without merging**